yarn.lock

```diff
@@ -5,46 +5,6 @@ __metadata:
   version: 6
   cacheKey: 8
 
-"@0no-co/graphql.web@npm:^1.0.1":
```

Comment:
Never commit yarn.lock if there are no changes in package.json. 

You can use `git checkout origin/main -- yarn.lock` to remove the changes :)

Reply:
i just resolved that yarn.lock issue , updated yarn.lock to original repo . thank you for your help :)